tsconfig.json

```diff
@@ -1,9 +1,15 @@
-/* Dev configuration */
 {
-  "extends": "./tsconfig.base.json",
-  "compilerOptions": {
-    "esModuleInterop": true, // Required fto remove warnings or jest test
-  },
-  /* During dev, src are included so they are also modified during refactorings. */
-  "include": ["src/**/*.ts", "test/**/*.ts"],
+    "compilerOptions": {
+        "incremental": true,
+        "target": "es2019",
+        "module": "commonjs",
```

Comment:
If this file is the base file for matter.js and node-matter, it should contain only the common parameters: it is more difficult to follow if parameters should be overridden.

Reply:
Yes I will clean this up